packages/fxa-shared/lib/sentry.ts

```diff
@@ -4,6 +4,7 @@
 
 import * as Sentry from '@sentry/browser';
 import { BrowserTracing } from '@sentry/tracing';
+import { SeverityLevel } from '@sentry/types';
```

Comment:
Q: Does [@sentry/types](https://www.npmjs.com/package/@sentry/types) need to be added to package.json?